x-pack/plugins/lens/readme.md

```diff
@@ -148,6 +148,8 @@ Example:
 }
 ```
 
+**Important!** To prevent conflicts, it's important to not re-use ad-hoc data view ids for different specs. If you change the spec in some way, make sure to also change its id. This even applies across multiple embeddables, sessions, etc. Ideally, the id will be globally unique. You can use the `uuid` package to generate a new unique id every time when you are changing the spec in some way. However, make sure to also not change the id on every single render either, as this will have a substantial performance impact.
```

Comment:
Would be nice to take this burden off the consumer somehow.

Reply:
agreed, but I don't have a good idea how... Open for suggestions!